internal/database/db.go

```diff
@@ -51,64 +52,57 @@ type FileInfo struct {
 var sqlOpen = sql.Open
 
 // NewDB creates a new DB connection
-func NewDB(c Pgconf) (*SQLdb, error) {
-	var err error
-
-	connInfo := buildConnInfo(c)
+func NewDB(config DBConf) (*SQLdb, error) {
+	connInfo := buildConnInfo(config)
 
 	log.Debugf("Connecting to DB with <%s>", connInfo)
 	db, err := sqlOpen("postgres", connInfo)
 	if err != nil {
-		log.Errorf("PostgresErrMsg 1: %s", err)
-		panic(err)
+		return nil, err
```

Comment:
Should we ad a retry here if the error is `connection refused`?

Reply:
We might, but I suggest doing it in a different PR if we decide to.